perception/navigator_lidar_oa/src/VolumeClassifier.h

```diff
@@ -11,51 +11,57 @@
 #include "ConnectedComponents.h"
 #include "navigator_msgs/PerceptionObject.h"
 #include <algorithm>
+#include "lidarParams.h"
 
 void VolumeClassifier(objectMessage &object)
 {
+	//Seperate variables we care about
 	auto h = object.maxHeightFromLidar;
 	auto x = object.scale.x;
 	auto y = object.scale.y;
 	auto z = object.scale.z;
-	//If we classify something as shooter/scanthecode, leave it alone
+
+	//Skip classification if the object is locked, set as a start_gate, or if it isn't real
 	if (object.locked || object.name == navigator_msgs::PerceptionObject::START_GATE_BUOY || !object.real) {
 		return;
 	}
 
+	//Possible classifications
 	std::vector<std::string> names = {"dock","shooter","scan_the_code","totem","buoy"};
-	double volumes[5][8] = { 	{0.7,	1.25,	6.0,	7.0,	6.0,	7.0,	2.5,	3.25}, //dock (NOT TESTED!)
```

Comment:
@maverick7170  

Can you put a comment about what these number means so that we can edit them in Hawaii?